install.sh

```diff
@@ -518,7 +518,7 @@ setup_selinux() {
         rpm_target=el7
         rpm_site_infix=centos/7
         package_installer=yum
-    elif [ "${VERSION_ID%%.*}" = "8" ]; then
+    elif [ "${VERSION_ID%%.*}" = "8" ] || [ "${VERSION_ID%%.*}" = "37" ]; then
```

Comment:
is Fedora 37 the only one that needs the EL8 packages, instead of EL7 or EL9?

Reply:
as far as I can see yes, it needs el8

Reply:
I recommend >= "37" so it can  work for future releases of fedora.

Reply:
Well I think 38 and newer need EL9 packages, and 36 needs EL7? I'm not sure, but some clarification would be good.